Sprint-1/exercises/initials.js

```diff
@@ -1,6 +1,9 @@
-let firstName = "Creola";
-let middleName = "Katherine";
-let lastName = "Johnson";
-
-// Declare a variable called initials that stores the first character of each string.
-// This should produce the string "CKJ", but you must not write the characters C, K, or J in the code of your solution.
+let firstName = "Creola";
+let middleName = "Katherine";
+let lastName = "Johnson";
+
+// Declare a variable called initials that stores the first character of each string.
+// This should produce the string "CKJ", but you must not write the characters C, K, or J in the code of your solution.
+
+let initial=firstName[0]+middleName[0]+lastName[0];

```

Comment:
Nice work here 😄 
How many initials do we have here? It looks like 3, right? But if I look at the variable name `initial` - it makes me think that there is only 1 initial. How can we change this to make it clear that there is more than one initial?